playground/AzureAppService/AzureAppService.AppHost/AzureAppService.AppHost.csproj

```diff
@@ -0,0 +1,25 @@
+<Project Sdk="Microsoft.NET.Sdk">
```

Comment:
Can you generate the manifest for this playground app?

Reply:
Done